README.md

```diff
@@ -1,13 +1,13 @@
 # Mentors Wiki
 
 ## Table of Contents:
-  - [Scheduling](sessions.md#phasely-scheduling)
-  - [Sessions](sessions.md)
-  - [What to do?](sessions.md#session-cadence)
+  - [Deputy](deputy.md)
+  - [Shifts](shifts.md)
+  - [Workday](workday.md)
+  - [Calendar](#calendar-overlay.md)
+  - [What to do?](shifts.md#during-a-shift)
```

Comment:
Somewhat redundant given it is only a few lines down in the shift.md file, but your call